src/Plugin/Filter/TokenFilter.php

```diff
@@ -89,7 +103,15 @@ public static function create(ContainerInterface $container, array $configuratio
    * {@inheritdoc}
    */
   public function process($text, $langcode) {
-    return new FilterProcessResult($this->token->replace($text), [], ['langcode' => $langcode]);
+    $data = [];
+
+    $entity = drupal_static('token_filter_entity', NULL);
```

Comment:
Because a filter isn't always rendered in a entity route, it can be rendered in a view that contains multiple entities.